src/main/java/com/bookstore/exception/EntityNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package com.bookstore.exception;
+
+public class EntityNotFoundException extends RuntimeException {
+    public EntityNotFoundException(String entity, Long id) {
+        super(entity + " not found with id = " + id);
```

Comment:
Interesting solution, I'll take note))
```suggestion
        super(String.format("%s not found with id = %s", entity, id));
```